lib/csv.rb

```diff
@@ -903,6 +918,21 @@ def to_csv(write_headers: true, **options)
     end
     alias_method :to_s, :to_csv
 
+    #
+    # Extracts the nested value specified by the sequence of name objects by calling dig at each step,
+    # returning nil if any intermediate step is nil.
+    #
+    def dig(index_or_header, *index_or_headers)
+      value = send(:[], index_or_header)
```

Comment:
`self[index_or_header` is better than `send(:[], index_or_header)`.

Reply:
Thanks, I fixed this part.